include/kickcat/Bus.h

```diff
@@ -149,6 +158,8 @@ namespace kickcat
 
         nanoseconds tiny_wait{200us};
         nanoseconds big_wait{10ms};
+
+        std::map<uint16_t, uint16_t> topology_;
```

Comment:
Add a short comment to describe what is associated here (parent id associated to slave id)